frontend/app/views/fragments/pricing/billingPeriodChoicePayPal.scala.html

```diff
@@ -0,0 +1,129 @@
+@import com.gu.salesforce.PaidTier
+@import com.gu.memsub.Current
+@import views.support.Pricing._
+@import com.gu.memsub.BillingPeriod._
+@import com.gu.memsub.subsv2.CatalogPlan.PaidMember
+@import com.gu.memsub.subsv2.MonthYearPlans
+
+@(plans: MonthYearPlans[PaidMember])
+
+
+<script type="text/javascript">
```

Comment:
Good question, my guess is that it relies on a lot of information that we have server-side only. @paulbrown1982 wrote most of this in [`billingPeriodChoice.scala.html`](https://github.com/guardian/membership-frontend/blob/master/frontend/app/views/fragments/pricing/billingPeriodChoice.scala.html), this file is just a temporary duplicate of that.